env/production/replica_count.yaml

```diff
@@ -64,3 +64,12 @@ metadata:
 spec:
```

Comment:
The production file will need to be added after bumping the tag

Reply:
thanks for the review and the catch - updated to remove